README.md

```diff
@@ -77,21 +77,26 @@ Set this to `tcd` if you want to only change the `pwd` for the current vim Tab.
 `update_on_change`:  Updates the current buffer to point to the new work tree if
 the file is found in the new project. Otherwise, the following command will be run.
 
-`update_on_change_command`: The vim command to run during the `update_on_change` event.
-Note, that this command will only be run when the current file is not found in the new worktree.
-This option defaults to `e .` which opens the root directory of the new worktree.
+`update_on_new_worktree`: The lua function to be run after changing the working
```

Comment:
I now realize that `update_on_worktree_change` may be more appropriate. I will update the PR.